src/components/index.js

```diff
@@ -0,0 +1,11 @@
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+import 'vuetify/dist/vuetify.min.css';
+import Upload from './Upload.vue';
+
+Vue.use(Vuetify);
+Vue.config.productionTip = false;
```

Comment:
I don't think this will actually work in all cases because you are importing `Upload` before registering vuetify.  At least, I had issues trying to do this in ResonantGeo, I don't remember the specifics.

The only way I found to register plugins automatically for consumers is to create a Vue plugin that does it on `install`, .e.g. <https://github.com/Kitware/resonantgeo/blob/master/src/index.js#L22>.

This could also be why `productionTip` isn't working for you.

Reply:
Hm, it worked fine for me with the UMD build, I was able to display the Upload component via the demo.html file shown above. Is it some kind of undefined behavior? Maybe based on when the templates get compiled?

Regarding the productionTip issue -- setting it to false here does work to remove the production tip, I just don't fully understand the behavior of the production tip. Am I supposed to have to add this line even when I'm building for production?

Reply:
I'm trying to remember the details of the problem I saw.  I'll update if I figure it out.

Going by how [vue uses productionTip](https://github.com/vuejs/vue/blob/52719ccab8fccffbdf497b96d3731dc86f04c1ce/src/platforms/web/runtime/index.js#L63), it shouldn't be necessary if you are building for production (`NODE_ENV='production'`).  It's only relevant if you want to suppress the messages in development.

Reply:
The UMD build is done in production mode (it even says so in the log output), and setting NODE_ENV for it doesn't change anything. I now see the source of the issue; the build on https://unpkg.com/vue is in development mode, hence the message. If I set it to https://unpkg.com/vue@2.5.17/dist/vue.min.js instead, it should be gone. I'll remove the productionTip directive from our code.

Reply:
Doing some excavation of ResonantGeo, I found the issue that I actually had was that the Vue object included in the component library was different than the one included in the App.  When doing `Vue.use(resonantgeo)`, it got a reference to the correct object.  This issue is mitigated by the fact that Vue is not bundled into redistributed package by the new CLI, so it's probably okay as is.